src/main/java/seedu/duke/task/Item.java

```diff
@@ -17,9 +18,11 @@
 
     protected String description;
     protected boolean isDone;
+    protected boolean isReturn;
     protected int priority;
     protected String category;
     protected LocalDate date;
+    protected LocalDate futureDate;
```

Comment:
Maybe it would be better if "isReturn" and "futureDate" is in Book.java (not a feature of the `Item` but of `Book`)